jre/guide/java/java_jre_configure_crypto_policy/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_all
+# platform = multi_platform_all,Java Runtime Environment
```

Comment:
Shouldn't the platform be only `Java Runtime Environment`? 
Do you keep it for testing reasons?
I have same comment for Bash and OVAL content.

Reply:
As the remediation content won't get anywhere without the respective rule, I don't think that it makes sense to restrict it in that way. The Ansible playbook will do its job on Fedora, and maybe on some other Linux distribution, so let's not restrict Ansible for the Java "platform".